### ISSUE TITLE
Crash 4.84 Rebug DEX

### ISSUE THREAD

**Issue by iMoD1998** [aldostools/webMAN-MOD#601]
Crash on 4.84 rebug DEX using mamba payload on XMB when trying to open PS3MAPI page.

To reproduce:
Use mamba payload 484D
boot up to xmb
go to /home.ps3mapi

This doesn't occur if you are in a game of any kinds, only the xmb.

**Comment by aldostools** [comment]
That page list all the ps3mapi modules in a single page.
You can narrow the module that is causing the bug accessing individually the following pages:

/led.ps3mapi
/buzzer.ps3mapi
/notify.ps3mapi
/getmem.ps3mapi
/setmem.ps3mapi
/syscall.ps3mapi
/syscall8.ps3mapi
/setidps.ps3mapi
/vshplugin.ps3mapi
/gameplugin.ps3mapi

It would help if you can capture the log using Mamba or Cobra debug

**Comment by iMoD1998** [comment]
/getmem.ps3mapi - crash
/setmem.ps3mapi - crash
/gameplugin.ps3mapi - crash

**Comment by iMoD1998** [comment]
also debug version of mamba seems to soft brick my console.

**Comment by aldostools** [comment]
Can you try with Cobra?

**Comment by iMoD1998** [comment]
With cobra, webman doesn load correctly probs because of issues with rebug.

**Comment by aldostools** [comment]
I will check the issue in the following days. Currently I'm not available to make changes.

About the issue with Cobra, I suggest that make a fresh installation of the CFW.
Remove /dev_hdd0/boot_plugins_dex.txt and reinstall the latest build webMAN.

**Comment by iMoD1998** [comment]
Yep, no problem its not a huge issue right now, like i said it only does it at XMB.

I have install firmware about 4 times via recovery, etc 

Webman just isn't compatible with the cobra thats in 4.84 drex.

I've had this happen on multiple DEX's 

Works fine with 4.84 on DECR though.
